doozer/doozerlib/backend/rebaser.py

```diff
@@ -90,19 +92,28 @@ def __init__(
         self._logger = logger or LOGGER
         self.rpm_lockfile_generator = RPMLockfileGenerator(runtime.repos, runtime=runtime)
         self.artifact_lockfile_generator = ArtifactLockfileGenerator(runtime=runtime)
+        self.image_repo = image_repo
+        self.uuid_tag = ''
 
         self.konflux_db = self._runtime.konflux_db
         if self.konflux_db:
             self.konflux_db.bind(KonfluxBuildRecord)
 
+        self.okd = okd
```

Comment:
Following on the Enum comment above.
```suggestion
        self.variant = variant
```